workspaces/src/worker/impls.rs

```diff
@@ -77,6 +77,10 @@ where
     ) -> ImportContractTransaction<'a, 'b> {
         self.workspace.import_contract(id, worker)
     }
+
+    async fn fast_forward(&self, delta_height: u64) -> anyhow::Result<()> {
```

Comment:
noting that adding this function to `StatePatcher` blurs that naming meaning. These things seem sandbox specific so not sure why the trait is needed yet